static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -312,6 +312,6 @@ const Wrapper = styled('ol')<{startLineNo: number}>`
   counter-reset: frame ${p => p.startLineNo - 1};
 
   && {
-    border-radius: 0;
+    border-radius: 0 !important;
```

Comment:
not a blocker but would it be possible to enforce this border radius without !important?

Reply:
I didn't write this line but found out it doesn't work unless we add !important. Turns out `border-radius: 5px` is being applied in one of the old group-detail.less files which I don't want to touch. 

![Screenshot 2023-02-10 at 11 34 21 AM](https://user-images.githubusercontent.com/4830259/218193944-60598510-a27c-4b18-a53a-87c8b4721d07.png)